frontend/src/metabase/components/EntityMenu.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import type { ComponentStory } from "@storybook/react";
+import EntityMenu from "./EntityMenu";
+
+export default {
+  title: "Entity Menu",
+  component: EntityMenu,
+};
+
+const Template: ComponentStory<typeof EntityMenu> = args => {
+  return <EntityMenu {...args} />;
+};
+
+const items = [
+  {
+    icon: "link",
+    title: "Option 1 - External link",
+    link: "https://google.com",
+    externalLink: true,
+  },
+  {
+    icon: "link",
+    title: "Option 2 - Relative link",
+    link: "/",
+  },
+  {
+    icon: "bolt",
+    title: "Option 3 - Action",
+    action: () => alert("Yo"),
+  },
+];
+
+export const Default = Template.bind({});
+Default.args = {
+  items,
+  trigger: <span>Hello</span>,
```

Comment:
As presented, it's not at alll obvious that you're supposed to click.

![Screen Shot 2023-03-02 at 3 14 37 PM](https://user-images.githubusercontent.com/30528226/222571275-8dc8d9d9-4c66-423a-bc48-ff3cd272443a.png)


```suggestion
  trigger: <span>Click Me</span>,
```